fe1-web/src/features/witness/components/WitnessNotification.tsx

```diff
@@ -19,13 +21,46 @@ import {
 } from '../objects/MessageToWitnessNotification';
 import { removeMessageToWitness } from '../reducer';
 
+const styles = StyleSheet.create({
+  container: {
+    padding: 20,
+    backgroundColor: Color.contrast,
+    borderRadius: 10,
+    shadowColor: Color.primary,
+    shadowOffset: { width: 0, height: 1 },
+    shadowOpacity: 0.05,
+    shadowRadius: 2,
+    elevation: 3,
+  } as ViewStyle,
+  marginB10: {
+    marginBottom: 10,
+  },
+  marginB15: {
+    marginBottom: 15,
+  },
+  marginT10: {
+    marginTop: 10,
```

Comment:
Is there a particular reason you use this instead of Spacing.x1, Spacing.x2 etc?